label_studio/core/views.py

```diff
@@ -203,16 +203,16 @@ def localfiles_data(request):
 
 
 def pachyderm_data(request):
-    """Serving files for LocalFilesImportStorage"""
+    """Serving files for PachydermImportStorage"""
     path = request.GET.get('d')
     redirect_url = request.GET.get('redirect')
 
-    if path and redirect_url and request.user.is_authenticated:
-        content_type, encoding = mimetypes.guess_type(str(path))
+    if redirect_url and request.user.is_authenticated:
+        content_type, _ = mimetypes.guess_type(str(path))
```

Comment:
You might be able to get away with using the redirect url here and then you don't need to use the path/`d` parameter at all.

Reply:
Good catch!